WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -482,9 +484,13 @@ class MySiteViewModel @Inject constructor(
     }
 
     fun refresh() {
-        selectedSiteRepository.updateSiteSettingsIfNecessary()
-        quickStartCardSource.refresh()
-        currentAvatarSource.refresh()
+        if (mySiteDashboardPhase2FeatureConfig.isEnabled()) {
+            mySiteSources.filterIsInstance(MySiteRefreshSource::class.java).forEach { it.refresh() }
+        } else {
+            selectedSiteRepository.updateSiteSettingsIfNecessary()
```

Comment:
Question (❓): Shouldn't this `selectedSiteRepository.updateSiteSettingsIfNecessary()` be called even if `mySiteDashboardPhase2FeatureConfig` is enabled? For example something like the below:

```
    fun refresh() {
        selectedSiteRepository.updateSiteSettingsIfNecessary()
        if (mySiteDashboardPhase2FeatureConfig.isEnabled()) {
            mySiteSources.filterIsInstance(MySiteRefreshSource::class.java).forEach { it.refresh() }
        } else {
            quickStartCardSource.refresh()
            currentAvatarSource.refresh()
        }
    }
```

Reply:
Nope, because it's part of refresh in `SelectedSiteSource`

Reply:
True, many thanks for the explanation Annmarie! 🙏